plugins/mcreator-core/blockly/js/mcreator_blocks.js

```diff
@@ -80,21 +80,29 @@ Blockly.Blocks['call_procedure'] = {
         this.setPreviousStatement(true);
         this.setNextStatement(true);
         this.setColour(250);
+        if (editorType == "procedures") {
+            Blockly.Extensions.apply('procedure_dependencies_tooltip', this, false);
+            Blockly.Extensions.apply('procedure_dependencies_mutator', this, true);
+            Blockly.Extensions.apply('procedure_dependencies_onchange_mixin', this, false);
+        }
     }
 };
 
-Blockly.Blocks['call_procedure_at'] = {
-    init: function () {
-        this.appendDummyInput()
-            .appendField(javabridge.t("blockly.block.call_procedure"))
-            .appendField(new FieldDataListSelector('procedure'), 'procedure')
-            .appendField(javabridge.t("blockly.block.call_procedure.at"));
-        this.appendValueInput('x').setCheck('Number').appendField('x: ');
-        this.appendValueInput('y').setCheck('Number').appendField('y: ');
-        this.appendValueInput('z').setCheck('Number').appendField('z: ');
-        this.setInputsInline(true);
+Blockly.Blocks['procedure_dependencies_mutator_container'] = {

```

Comment:
These definitions could be replaced with helper functions (similar to feature extension blocks)

Reply:
:heavy_check_mark: Fixed